resilio-sync-np@2.8.1.1390: download via aria2 failed

Installing 'resilio-sync-np' (2.8.1.1390) [64bit] from 'nonportable' bucket
Starting download with aria2 ...
Download: 11/29 21:13:19 [ERROR] CUID#8 - Download aborted. URI=https://download-cdn.resilio.com/stable/windows64/ResiliDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://download-cdn.resilio.com/stable/windows64/Resilio-Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 08ec3a|ERR |       0B/s|I:/Scoop/cache/resilio-sync-np#2.8.1.1390#92fc9e9.exe
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

WARN  Download failed! (Error 3) Resource was not found
WARN  https://download-cdn.resilio.com/stable/windows64/Resilio-Sync_x64.exe#/setup.exe
    referer=https://download-cdn.resilio.com/stable/windows64/Resilio-Sync_x64.exe#/
    dir=I:\Scoop\cache
    out=resilio-sync-np#2.8.1.1390#92fc9e9.exe

--- a/bucket/resilio-sync-np.json
+++ b/bucket/resilio-sync-np.json
@@ -25,7 +25,9 @@
         "script": [
             "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
             "$cec = @{1 = 'You may need to restart your PC for the app to be completely removed'}",
-            "Invoke-ExternalCommand \"$Env:AppData\\Resilio Sync\\Resilio Sync.exe\" -ArgumentList @('/UNINSTALL', '/S', '/REMSETTINGS') -ContinueExitCodes $cec"
+            "$args = @('/UNINSTALL', '/S')",
+            "if ($cmd -ne 'update') { $args += '/REMSETTINGS' }",
+            "Invoke-ExternalCommand \"$Env:AppData\\Resilio Sync\\Resilio Sync.exe\" -ArgumentList $args -ContinueExitCodes $cec"
         ]
     },
     "checkver": {

